### DISCUSSION TITLE
【PC端问题】1.1.14+1规则导入失败

### DISCUSSION THREAD

**Discussion by Holyxiea** [xiaohucode/yidaRule#195] [Q&A]
![微信截图_20240715085917](https://github.com/user-attachments/assets/778215b7-f5ca-459b-8222-9766d67d596a)
如上图，开了代理也是一样提示

**Comment by Luke2084** [comment]


**Reply by Oldsport-996** [reply]
1.1.27版本还是不行 PC没法导入（那个最新的压缩包解压出来的是27版本）

**Reply by fuxin-mail** [reply]
> 1.1.27版本还是不行 PC没法导入（那个最新的压缩包解压出来的是27版本）

你的问题解决了吗？我也是这样

**Reply by Oldsport-996** [reply]
解决不了啊，PC软件也没找到可以本地导入的地方，网络一直都导不进去，我用的就是官方那个链接

**Reply by Oldsport-996** [reply]
代理加速器啥的都开了 全局 分流节点 都试了

**Reply by maomaovip** [reply]
我的 ios 也是这样的，不知道怎么解决

**Comment by falling7down** [comment]


**Reply by fuxin-mail** [reply]
![PixPin_2024-08-12_20-19-18](https://github.com/user-attachments/assets/2b112f7a-f7c3-42ff-b388-6f0226cb29b5)
规则网址不是粘贴到1位置吗？我粘贴了右上角导入是灰色不可点，也不显示成功呢？2位置是打开软件就自动显示这一个网址，点一下这个就自动填到1了，这个还无法替换，我看着和仓库那个规则地址还不一样。

**Reply by fuxin-mail** [reply]
对现在就是下载了导入的

**Reply by fuxin-mail** [reply]
这就奇怪了，就是开始用的时候成功导入过一次，后来总是导入不了

**Reply by fuxin-mail** [reply]
我一直开着github加速和代理也不行

**Comment by G0heng** [comment]
网络导入好像就是不行，浏览器访问那个规则链接，把yds文件下载电脑上，然后用本地导入

**Reply by fuxin-mail** [reply]
多点几次我的可以了
